Problems/Set-1/1/main.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+using namespace std;
+#define HEIGHT 5
+#define WIDTH 3
+
+#define XHEIGHT 1
+#define XWIDTH 1
+
+#define YHEIGHT 2
+#define YWIDTH 2
+
+//Assumptions. 
+// width=height on windows (screen is parallelogram)
+//complexity 0(n) 
+//proof to follow soon
+void solve(){
+	int x,y,min_screen,extra_screen,remainder_x_blocks,x_screens,screens;
+	cin>>x;
+	cin>>y;
+	min_screen = (int)(y/2);			//at least those screens due to big blocks
+	extra_screen = ((bool)(y%2));		//in case we have leftover big block
```

Comment:
You don't need to convert it to bool (why?).

Reply:
Thats correct, i was probably tired :P